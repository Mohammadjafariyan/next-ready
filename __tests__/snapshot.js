import { render } from '@testing-library/react'
import Page from '../app/sample'
 
it('renders homepage unchanged', () => {
  const { container } = render(<Page />)
  
  expect(container).toMatchSnapshot()
})