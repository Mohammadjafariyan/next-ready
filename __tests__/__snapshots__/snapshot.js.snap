// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    id="hi"
  >
    hi
  </div>
</div>
`;
